src/devtools/packages/devtools-reps/object-inspector/components/ObjectInspector.tsx

```diff
@@ -76,15 +79,14 @@ class OI extends PureComponent<ObjectInspectorProps> {
     this.ensureRootsLoaded();
   }
 
-  // TODO [hbenl] this should not be necessary: the ObjectInspector should only
+  // this should not be necessary: the ObjectInspector should only
   // be rendered with a root ValueItem that has already been loaded.
-  // However currently (at least) the console doesn't ensure that this is the case,
-  // so we need this workaround for now.
```

Comment:
Curious why we're removing the TODO part of this comment but leaving the "_this should not be necessary_..." bit?

Is the implied todo here that we'll remove it later so long as we don't see "devtools-oi-root-not-loaded" events being logged? (Maybe we can spell that out more clearly for someone like me, if so?)

Reply:
The TODO was intended for fixing the known console issue and adding telemetry. So this entire workaround should not be necessary anymore but I thought we could keep it to make the ObjectInspector resilient in case it is used incorrectly again.